DataAccess/Repositories/TargetAudienceRepository.cs

```diff
@@ -67,5 +67,31 @@ public List<TargetAudienceProjection> GetAllWithTestAmount()
             return result;
         }
     }
+    public List<TargetAudience> GetAllActiveWithTest()
+    {
+        using (Repository context = new Repository())
```

Comment:
Dit klopt alleen niet (overigens gebruik je nu op alleen deze plek de query vorm van het ophalen van de data).

Je moet alle targetaudiences ophalen met een 1 actieve test. Nu haal je alle targetaudiences op met een test. (inactief of niet)